backend/config.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
     },
   },
   notifications: {
+    enabled: process.env.NOTIFY_ENABLED ? process.env.NOTIFY_ENABLED === 'true' : true,
```

Comment:
As a first pass to make sure things still work in dev/pre/prod. Will set up env vars properly as a separate PR.